### PR TITLE
updated to kafka 0.10

### PR DESCRIPTION
**Pull request by xelax**
I did the minimal changed needed to update to kafka 0.10 and pass the test, plus a workaround for the wrong version of kakfa being pulled in by embedded-kafka

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @acozziebay,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by typesafehub-validator** [comment]
Hi @acozziebay,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: core/src/main/scala/akka/kafka/KafkaConsumerActor.scala [hunk 1]
```diff
@@ -153,8 +153,8 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
       //set partitions to fetch
       val partitionsToFetch = requests.keys.toSet
       consumer.assignment().asScala.foreach { tp =>
-        if (partitionsToFetch.contains(tp)) consumer.resume(tp)
-        else consumer.pause(tp)
+        if (partitionsToFetch.contains(tp)) consumer.resume(Seq(tp).asJava)
+        else consumer.pause(Seq(tp).asJava)
```
Comment (13h3r) [inline core/src/main/scala/akka/kafka/KafkaConsumerActor.scala]: Could we optimize this somehow to not calling each partition if `Seq` API is available? If it is easy my suggestion is to do it in this pr

**Comment by patriknw** [comment]
@xelax We would like to get this in asap and release a new milestone. Would you be able to sign the Lightbend CLA online at http://www.lightbend.com/contribute/cla
Thanks for contributing!

**Comment by xelax** [comment]
Hi Patrick: I'd like to get it signed but I need to have the corporate CLA version signed, which might take a couple of days. 
I'd hate to block the new release: feel free to reject the pull request and do the changes yourselves, they are simple enough. I am sure I will soon have other opportunities to contribute as soon as the CLA is signed.

**Comment by 13h3r** [comment]
Thanks @xelax. I will close this PR and do the changes.

**Pull request closed without merging**